assets/basic/1.0/css/style.css

```diff
@@ -3,6 +3,7 @@
 
 /* Modular scale, colors, etc. */
 @import url("lib/variables.css");
+@import url("lib/gc-design-system-variable-proposal.css");
```

Comment:
@duboisp I think you should have a proposal for a theme integration, unless you want to make it GC exclusive. Specifically if you already have the font family declared in font.css instead of design-system-variables-proposal.css.

Reply:
I just merged and integrated previous contribution.

That variable proposal was based on Josh previous work. See PR #14